File: official/docs/php/batches/CreateLabelForBatch.php
<?php

require_once("../lib/easypost.php");
\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");

$batch = \EasyPost\Batch::retrieve('batch_...');

$batch->label(array('file_format' => 'PDF'));

echo $batch;

File: official/docs/go/batches/create_label_for_batch.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	batch, err := client.GetBatchLabels("batch_...", "PDF")

	fmt.Println(batch) 
}
